src/mono/wasm/debugger/BrowserDebugProxy/MemberObjectsExplorer.cs

```diff
@@ -454,7 +461,7 @@ async Task UpdateBackingFieldWithPropertyAttributes(JObject backingField, string
             {
                 string returnTypeName = await sdbHelper.GetReturnType(getMethodId, token);
                 JObject propRet = null;
-                if (isAutoExpandable || (state is DebuggerBrowsableState.RootHidden && IsACollectionType(returnTypeName)))
+                if (getCommandOptions.HasFlag(GetObjectCommandOptions.AutoExpandable) || getCommandOptions.HasFlag(GetObjectCommandOptions.ForDebuggerProxyAttribute) || (state is DebuggerBrowsableState.RootHidden && IsACollectionType(returnTypeName)))
```

Comment:
The original code was passing `isAutoExpandable: getCommandOptions.HasFlag(GetObjectCommandOptions.ForDebuggerProxyAttribute)`, but here we are adding `HasFlag(AutoExpandable)` also. Is that deliberate? If we just want to match the old code then I would keep `isAutoExpandable`, and add `getCommandOptions` to the parameters, so the caller keeps control of when to treat something as autoexpandable.

Reply:
Oh, sorry, I just saw that `AutoExpandable` is new. Ignore that comment above.